models/recruit/src/index.ts

```diff
@@ -135,7 +135,7 @@ export class TCandidate extends TPerson implements Candidate {
 }
 
 @Mixin(recruit.mixin.VacancyList, contact.class.Organization)
-@UX(recruit.string.VacancyList, recruit.icon.RecruitApplication, undefined, 'name')
+@UX(recruit.string.VacancyList, recruit.icon.RecruitApplication, 'ORG', 'name')
```

Comment:
```suggestion
@UX(recruit.string.VacancyList, recruit.icon.RecruitApplication, 'CM', 'name')
```

We renamed ORG to Company, so let's think about different name.